swagger.yaml

```diff
@@ -3754,6 +3764,21 @@ definitions:
       console_url:
         type: string
 
+  disk-config-params:
+    type: object
+    required:
+      - 'id'
+      - 'path'
+    properties:
+      id:
+        type: string
+      path:
+        type: string
+      role:
```

Comment:
That role will be used in other places as well so better put it under definitions globally

Reply:
Done